std/split/split_gl.asm

```diff
@@ -22,9 +22,11 @@ machine WrapGL(RESET, operation_id) {
 
     // 2. Build the output, packing the least significant 4 byte into
     //    a field element
-    col witness output;
-    col fixed FACTOR_OUTPUT = [0x100, 0x10000, 0x1000000, 0, 0, 0, 0, 1]*;
-    output' = (1 - RESET) * output + bytes * FACTOR_OUTPUT;
+    col witness output_low, output_high;
+    col fixed FACTOR_OUTPUT_LOW = [0x100, 0x10000, 0x1000000, 0, 0, 0, 0, 1]*;
+    col fixed FACTOR_OUTPUT_HIGH = [0, 0, 0, 1, 0x100, 0x10000, 0x1000000, 0]*;
+    output_low' = (1 - RESET) * output_low + bytes * FACTOR_OUTPUT_LOW;
```

Comment:
so bytes here is the full 8 bytes, right?

Reply:
Every entry in the bytes column is one byte, but there are 8 rows in a block, so it lists the full 8-byte decomposition of the input.

Reply:
ok